sdk/python/Makefile

```diff
@@ -23,9 +23,12 @@ ensure:: .make/ensure/uv
 build_package:: ensure
 	uv run -m build --outdir ./build --installer uv
 
-build_plugin::
+build_plugin: ../../bin/pulumi-language-python
+
+.PHONY: ../../bin/pulumi-language-python
```

Comment:
Hrm shouldn't `build_plugin` be `.PHONY` rather than this?  `../../bin/pulumi-language-python` is an actual output of the Makefile.

Reply:
It's a actual output, but it isn't determined by its inputs, so make doesn't understand when it needs to be rebuilt. It needs to be phony.